src/pages/app/p/[package].settings.vue

```
@@ -86,6 +91,98 @@ async function didCancel(name: string) {
 }
 
+async function setUpdateChannelSync(value: boolean) {
+  if (value === true && appRef.value && (appRef.value.default_channel_ios as any).id !== (appRef.value.default_channel_android as any).id) {
+    const channels = await supabase
+      .from('channels')
+      .select('*')
+      .eq('app_id', appId.value)
+      .in('id', [
+        ...(appRef.value.default_channel_ios ? [(appRef.value.default_channel_ios as any).id] : []),
+        ...(appRef.value.default_channel_android ? [(appRef.value.default_channel_android as any).id] : [])
+      ])
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)